main/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
```

Comment:
For convenience, you can add functionality to the admin panel

Reply:
Ok, I will add some  functionality to the admin panel.